core/tests/unit/ldml/meson.build

```
@@ -94,14 +120,34 @@ if cpp_compiler.get_id() == 'emscripten'
   normalization_tests_flags += ['-lnodefs.js', '-sEXPORTED_RUNTIME_METHODS=[\'UTF8ToString\']']
 endif
 
-t = executable('test_context_normalization',
+tc = executable('test_context_normalization',
```

```suggestion
test_context_normalization = executable('test_context_normalization',
```

Can we use the full process name as the variable name? Makes future maintenance so much easier